global_helpers/panther_oss_helpers.py

```diff
@@ -235,6 +237,32 @@ def set_key_expiration(key: str, epoch_seconds: int) -> None:
     )
 
 
+def put_dictionary(key: str, val: dict, epoch_seconds: int = None):
+    # Convert the data dictionary to a JSON string
+    data = json.dumps(val)
+
+    # Store the item in DynamoDB
+    kv_table().put_item(Item={"key": key, "data": {"S": data}})
```

Comment:
Rather than hard-coding the column name here as data, let's add a new global like we do for string set and counter/int https://github.com/panther-labs/panther-analysis/blob/a9554946a5d76bfda7eebfef1f706b54d0530c2f/global_helpers/panther_oss_helpers.py#L171-L172

Reply:
Done!